internal/common/util/list.go

```diff
@@ -45,3 +45,16 @@ func Filter[T any](list []T, predicate func(val T) bool) []T {
 	}
 	return out
 }
+
+func Concat[T any](slices [][]T) []T {
```

Comment:
I think there's a case for this to be `Concat[T any](slices ...[]T) []T`  I think the current function sig is a bit more like a `flatmap` (albeit without the `map`)!